packages/kotti-ui/source/kotti-button/KtButton.vue

```diff
@@ -8,11 +8,12 @@
 			@click="handleClick"
 		>
 			<i v-if="isLoading" class="kt-circle-loading" />
-			<i v-else-if="icon !== null" class="yoco" v-text="icon" />
+			<i v-else-if="hasIconLeft" class="yoco" v-text="icon" />
 			<span v-if="hasSlot">
 				<slot />
 			</span>
 			<span v-else-if="label !== null" v-text="label" />
+			<i v-if="hasIconRight && !isLoading" class="yoco" v-text="icon" />
```

Comment:
technically for the left Icon we remove the icon and replace it with the loading icon (loading comes on left)

but you don't need the `!isLoading` for the right Icon

Reply:
@carsoli is it agreed that loading is rendered always on the left side of the button regardless of the icon position?
or should it be placed in the same position as the icon?

Reply:
Good point @santiagoballadares probably this is a question for Alberto.

Reply:
@santiagoballadares just asked Alberto he agreed to keep it on the left.